static/docs/user-guide/contributing.md

```diff
@@ -288,11 +288,11 @@ Fixes #(Github issue id).
 Message types:
 
 - *component* - name of a component that this patch is affecting. Use `dvc` in a
-  general case;
-- _short description_ - short description of the patch;
+  general case.
+- _short description_ - short description of the patch.
 - _long description_ - If needed, longer message describing the patch in more
-  details;
-- _github issue id_ - An id of the Github issue that this patch is addressing
+  details.
+- _github issue id_ - An id of the Github issue that this patch is addressing.
```

Comment:
Start bullet this (lower case) text: `id of the Github...` (just remove `An`). No period . at the end.